.github/workflows/aws-main.yml

```diff
@@ -187,8 +187,8 @@ jobs:
   push:
     name: "Push images"
     runs-on: ubuntu-latest
-    # push image on master, target branch not set, and the dependent steps were either successful or skipped
```

Comment:
Small heads up that this (and all other locations here that check for the main branch) will need to rebase the changes added in #12829